packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/parse-file-from-string.ts

```diff
@@ -14,11 +14,18 @@ type ParsedFile = {
 export function parseFileFromString(fileAsString: string): ParsedFile {
   let decodedBytes: Buffer;
 
-  try {
-    decodedBytes = Buffer.from(fileAsString, "base64");
-  } catch (ex) {
-    log("Got a non-base64 document! Using original fileAsString content for decodedBytes");
+  // with MTOM we now have non b64 encoded files in the body of the response.
+  const firstSixBytesBuffer = Buffer.from(fileAsString.slice(0, 6));
+  if (isXMLContentType(firstSixBytesBuffer)) {
```

Comment:
With MTOM we now have non b64 encoded files in the body of the response. For xml files, those don't trigger the try catch below, and need a check here to see if they are xmls.